src/scala/microsoft-spark-3.0.x/src/main/scala/org/apache/spark/sql/api/dotnet/SQLUtils.scala

```diff
@@ -35,4 +36,15 @@ object SQLUtils {
       broadcastVars,
       accumulator)
   }
+
+  /**
+   * Wrap `SparkContext.broadcast` since it involves ClassTag.
+   * TODO: Investigate to see `DotnetBackendHandler` can be updated to support functions
+   * that require ClassTags.
+   */
+  def createBroadcast(
+      context: SparkContext,
+      pythonBroadcast: PythonBroadcast): Broadcast[PythonBroadcast] = {
+    context.broadcast(pythonBroadcast)
+  }
```

Comment:
Is this necessary ?   Why not call the `JavaSparkContext.Broadcast` method.

Reply:
I just followed the pyspark flow and that creates a PythonBroadcast object.

Reply:
Have you tried calling `JavaSparkContext.Broadcast` instead of relying on this extra method ?

Reply:
Good point. Since we are already converting to `JavaSparkContext` inside `BroadcastVariable` anyway, we can first convert to Java version and use it.

Reply:
Sounds good, will do.